projects/storefrontlib/src/lib/store-finder/store/selectors/find-stores.selectors.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { TestBed } from '@angular/core/testing';
+import { StoreModule, Store, combineReducers, select } from '@ngrx/store';
+
+import { SearchConfig } from '../../models/search-config';
+
+import * as fromRoot from '../../../routing/store';
+import * as fromReducers from '../reducers';
+import * as fromActions from '../actions';
+import * as fromSelectors from './find-stores.selectors';
+
+describe('FindStores Selectors', () => {
+  let store: Store<fromReducers.StoresState>;
+
+  const searchResult = { stores: [{ name: 'test' }] };
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({
+          ...fromRoot.getReducers(),
+          stores: combineReducers(fromReducers.reducers)
+        })
+      ]
+    });
+    store = TestBed.get(Store);
+    spyOn(store, 'dispatch').and.callThrough();
+  });
+
+  describe('findStores', () => {
+    it('should return the stores search results', () => {
+      let result;
+      const searchConfig: SearchConfig = { pageSize: 10 };
+      store
+        .pipe(select(fromSelectors.getFindStoresEntities))
+        .subscribe(value => (result = value));
+
+      expect(result).toEqual({});
+
+      store.dispatch(
+        new fromActions.FindStores({ queryText: 'test', searchConfig })
```

Comment:
I don't think this is required since you are really only checking the result of the success action which you are dispatching right after.